source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,200 @@
+
+.. code-block:: bash
+
+	10.0.0.0 - 10.255.255.255 (10/8)
+	172.16.0.0 - 172.31.255.255 (172.16/12)
+	192.168.0.0 - 192.168.255.255 (192.168/16)
+
+To find the IP address of the machine that is running Unbound, we use:
+
+.. code-block:: bash
+
+	hostname --all-ip-addresses
+
+If you just have one IP address as output from the :command:`hostname` command that will be the correct one. If you have multiple IP addresses the easiest way which IP address to use, is to find out which connection goes to your home router. Keep in mind that finding the wrong IP address here this can be a source of connectivity errors further on. For purpose of this tutorial we imagine that our home router has ``10.0.0.1`` as IP address, and our resolver machine (the machine that is running our Unbound instance) has ``10.0.0.2``, which we will get into in the next section.
+
+As prerequisite for the next step we need to configure our Unbound instance to be reachable from other devices than only the machine on which the instance is running. The full example config is almost 1200 lines long, as the capabilities of Unbound are considerable, but we won’t need nearly as much. (If you are interested, any and all configurables can be found in the extensive manual page with :manpage:`unbound.conf`.
+
+The example config is found at:
+
+.. code-block:: bash
+
+	/etc/unbound/unbound.conf
+
+If you open this for the first time it looks very empty. It is still usable for one machine, as this is how the Unbound defaults are configured. It's not, however, enough for our purposes, so we will add the minimal configuration options needed.
+
+The options that we add to the current config file to make it a "minimal usable config" are as follows. Note that the IPv6 options are commented out, but we recommend to uncomment them if your router and network supports it.
+
+.. code-block:: bash
+
+	server:
+			# location of the trust anchor file that enables DNSSEC
+			auto-trust-anchor-file: "/var/lib/unbound/root.key"
+			# the interface that is used to connect to the network, this means on this machine
+			interface: 0.0.0.0
+			# interface: ::0
+			# addresses from the IP range that are allowed to connect to the resolver
+			access-control: 10.0.0.0/8 allow
+			# access-control: 2001:DB8.. code-block:: bash/64 allow
+	remote-control:
+			control-enable: yes
+
+The access-control is currently configured to listen to any address on the machine, and only allow queries from the ``10.0.0.0/8`` `IP subnet <https://www.ripe.net/about-us/press-centre/understanding-ip-addressing>`_ range. Note that the IP addresses we chose (``10.0.0.1`` and ``10.0.0.2``) fall within the ``10.0.0.0/8`` range.
```

Comment:
I suspect that access control doesn't define where Unbound listens, that is determined by interface surely?

s/Note that the IP address we chose/Note that the IP address we chose above/